### PR TITLE
better control of build optimization flags for native module

### PR DESCRIPTION
**Pull request by mistafunk**

### CODE REVIEW

**Comment by clechot** [comment]
The --debug flag works with the setup.py build command. Could you add something in the documentation about how to apply that to PyPRT common commands (for ex. `pip install -e .`, `python setup.py bdist_wheel`)?

**Review by clechot — CHANGES_REQUESTED** [review]
See above comment

**Comment by mistafunk** [comment]
Thanks for the review. Good catch regarding the docs. I've extended the developer documentation section.

**Review by clechot — CHANGES_REQUESTED** [review]
The `PIP_INSTALL_OPTION` argument doesn't seem to work on Windows.

File: README.md [hunk 1]
```diff
@@ -129,6 +129,16 @@ The workflow is the same as iterating on the Python files but needs additional s
 This will allow you to run `ninja install` after changes on the C++ files and update the "live" PyPRT package in the `pyprt` subdir.
 The `setup.py clean` call mentioned above will also clean out the native extension from the `pyprt` source directory.
 
+#### C++ Debug Builds
+
+By default, the native module is built in "release" mode (full optimization, no debug symbols). If the `--debug` argument is passed to `setup.py`, the CMake scripts will switch to "Release with Debug Info" mode, this will _disable optimization_ (`-O0`) to provide correct per-line debugging.
+
+To enable native debugging when building wheels or conda packages use:
+`python setup.py build --debug bdist_wheel` (or `bdist_conda`)
```
Comment (clechot) [inline README.md:137]: As discussed, we need to add the "--skip-build" argument as well to avoid running the build command twice.

Comment (mistafunk) [reply README.md]: done

**Comment by mistafunk** [comment]
> The `PIP_INSTALL_OPTION` argument doesn't seem to work on Windows.

fixed by using an alternate syntax

**Review by clechot — APPROVED** [review]

**Pull request merged**